packages/venia-concept/src/index.js

```diff
@@ -23,17 +23,4 @@ ReactDOM.render(
     document.getElementById('root')
 );
 
-if (process.env.SERVICE_WORKER && 'serviceWorker' in navigator) {
```

Comment:
This change seems unrelated.

Reply:
Good catch. This shouldn't have been included in the PR.

Reply:
Fixed. 👍